lamp/create-group.php

```diff
@@ -72,5 +72,14 @@ function render_create_group_page($error=null) { ?>
             if (!validate_input($_POST, "group_name")) return render_create_group_page("Please provide a group name");
 
             $group_name = parse_input('group_name', true);
-            Channel::create_channel($course->course_id, $group_name, 2);
+            $group = Channel::create_channel($course->course_id, $group_name, 2);
+            GroupMembership::create_membership($_SESSION['uid'], $group->ch_id);
+            foreach ($users_in_course as $user)
+            {
```

Comment:
Bug: you applied a database optimization in commit 11d624, by requesting users after checking if the user is an instructor. Because of that change, $users_in_course is no longer defined at this point, so users are not added to courses.

Can you add this to the Jira so we remember to fix it?

@cengman2017